module/os/linux/zfs/zfs_dir.c

```diff
@@ -416,6 +417,7 @@ zfs_dirlook(znode_t *dzp, char *name, znode_t **zpp, int flags,
 		rw_exit(&dzp->z_parent_lock);
 	} else if (zfs_has_ctldir(dzp) && strcmp(name, ZFS_CTLDIR_NAME) == 0) {
 		ip = zfsctl_root(dzp);
+		VERIFY3P(ip, !=, NULL);
```

Comment:
```suggestion
		ASSERT3P(ip, !=, NULL);
```

Perhaps better to just ASSERT in this particular case.